Documentation/Signoffs/Main_Processor.md

```diff
@@ -1,53 +1,80 @@
-## How to submit a signoff or update a signoff:
+# Processing Subsystem
 
-### Submitting Signoff
-1. Create the signoff markdown file 
-2. At the bottom of the page select "create a new branch for this commit and start a pull request"
-3. Submit the changes 
-4. Name the pull request branch "name-signoff-subsystem name"
-5. The comment should be "requesting signoff for name of subsystem"
-6. Submit the pull request
-7. Wait for faculty supervisor feedback on the pull request
+## Function of the Subsystem
 
-### Updating Signoff
+<img width="728" alt="Screen Shot 2023-04-04 at 3 25 34 PM" src="https://user-images.githubusercontent.com/123997954/229912228-53e792c0-7cce-496a-ace8-6fa46197ea0d.png">
+The goal of this subsystem is to process the input signal, and output an appropriate response to the ambient noise present in the room. The processing subsystem consists of an ADC converter which will take the input and convert it into digital samples which will be analyzed and processed by the adaptive filter algorithms. The output will then be converted using a DAC converter and outputted to the other parts of the system.
 
-1. Edit the signoff markdown file 
-2. Add a description of how you have addressed the feedback from the faculty supervisor
-3. At the bottom of the page select "commit directly to"
-4. Submit the changes 
+## Constraints
 
-Making tables for constraints and BOM will use Excel to Markdown here:
-https://tabletomarkdown.com/convert-spreadsheet-to-markdown/
+| No. | Specifications and Constraints                     | Origin          |
+| --- | -------------------------------------------------- | --------------- |
+| 1   | Shall be powered from wall outlet                  | Specification   |
+| 2   | Shall process and output information within 1.4 ms | Specification   |
+| 3   | Shall process sounds at a rate of at least 40 kHz  | Specification   |
+| 4   | Shall keep track of outputs reaching 80+ dB        | Greater Impacts |
+| 5   | Shall minimize storage of classroom noise          | Ethics          |
+| 6   | Shall have a safe power toggle                     | IEC 60950-1     |
+| 7   | Shall not reach 60°C (140°F)                       | Safety          |
```

Comment:
How will this be ensured?

Reply:
Updated constraint table, will add sections for each constraint